rslib/src/cloze.rs

```diff
@@ -57,26 +57,30 @@ pub fn reveal_cloze_text(text: &str, cloze_ord: u16, question: bool) -> Cow<str>
             .parse()
             .unwrap_or(0);
 
+        let text = caps.get(cloze_caps::TEXT).unwrap().as_str().to_owned();
         if captured_ord != cloze_ord {
             // other cloze deletions are unchanged
-            return caps.get(cloze_caps::TEXT).unwrap().as_str().to_owned();
+            return text;
         } else {
             cloze_ord_was_in_text = true;
         }
 
+        let text_attr;
         let replacement;
         if question {
+            text_attr = format!(r#" data-cloze="{}""#, htmlescape::encode_minimal(&text));
```

Comment:
minimal is not appropriate here: https://docs.rs/htmlescape/latest/htmlescape/fn.encode_minimal.html#safety-notes

Reply:
Alright, then I'll create a helper function for the pylib tests. `html.escape` doesn't encode as thoroughly as Rust's `htmlescape::encode_attribute`:)